pom.xml

```diff
@@ -18,16 +18,16 @@
   <parent>
     <groupId>com.arpnetworking.build</groupId>
     <artifactId>arpnetworking-parent-pom</artifactId>
-    <version>1.0.27</version>
+    <version>1.0.28</version>
     <relativePath />
   </parent>
 
   <modelVersion>4.0.0</modelVersion>
-  <groupId>com.arpnetworking.metrics</groupId>
-  <artifactId>sink-http-apache</artifactId>
+  <groupId>com.arpnetworking.metrics.client</groupId>
```

Comment:
It's a prerelease version that was never reviewed. This is acceptable. Happy discuss how it's released as per #1.